src/pages/article.js

```diff
@@ -4,13 +4,17 @@ import styled from 'styled-components';
 
 import Layout from '../components/Layout';
 import useFetch from '../hooks/useFetch';
+import {dateFromID} from '../utils/unit';
 
 function showPost(data) {
 	return (
 		<>
 			<Title>{data.title}</Title>
 			<Author>von : {data.author}</Author>
-			<Published>veröffentlicht am: {Date(data.published)}</Published>
+			<Published>
+				veröffentlicht am: {dateFromID(data._id).toLocaleDateString()} um:{' '}
+				{dateFromID(data._id).toLocaleTimeString()}
+			</Published>{' '}
```

Comment:
Do you really need this string with just a space in it?

Reply:
no, that's just some automatic stuff react keeps adding in and sometimes I just didn't thought of them